sql/analyzer/index_analyzer_test.go

```diff
@@ -65,11 +65,13 @@ func TestMatchingIndexes(t *testing.T) {
 
 	require.Equal(t, []sql.Index{dummy1, dummy2, dummy3}, ia.MatchingIndexes(ctx, testDb, testTable, v1))
```

Comment:
This could use some more unit test for `MatchingIndex` now that the logic has solidified a bit